x-pack/test/security_solution_api_integration/test_suites/detections_response/utils/rules/create_non_security_rule.ts

```diff
@@ -0,0 +1,41 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import type SuperTest from 'supertest';
+
+const SIMPLE_APM_RULE_DATA = {
+  name: 'Test rule',
+  rule_type_id: 'apm.anomaly',
+  enabled: false,
+  consumer: 'alerts',
+  tags: [],
+  actions: [],
+  params: {
+    windowSize: 30,
+    windowUnit: 'm',
+    anomalySeverityType: 'critical',
+    environment: 'ENVIRONMENT_ALL',
+  },
+  schedule: {
+    interval: '10m',
+  },
+};
+
+/**
+ * Created a non security rule. Helpful in tests to verify functionality works with presence of non security rules.
+ * @param supertest The supertest deps
+ */
+export async function createNonSecurityRule(
```

Comment:
I was going to ask if alerting provided some helpers for this, but discovered that several (at least 3) other test suites have similar code to exercise this API themselves. So: nothing needed, for now.

I do wonder whether it would be a useful effort to spend some time building out solution-based ["services"](https://github.com/elastic/kibana/blob/main/x-pack/test/api_integration/services/index.ts) for FTRs.